starboard/shared/starboard/application.h

```diff
@@ -41,7 +41,7 @@ namespace starboard {
 
 // A small application framework for managing the application life-cycle, and
 // dispatching events to the Starboard event handler, SbEventHandle.
-class Application {
+class __attribute__((visibility("default"))) Application {
```

Comment:
@y4vor I'm unsure if it's okay to add SB_EXPORT to these non-Sb symbols that are needed for nplb